tests/test_schema_errors/test_integers.py

```diff
@@ -0,0 +1,94 @@
+import pytest
```

Comment:
this is a good test in general. Still, I would put this as part of the schema tester tests. You seem to displike large test files, but really below 500 lines a test file is ok in my book.

Reply:
I structured this mostly, to easily progress through the data types docs and put tests in place for the different format validation, etc. that we want to support. 

Now that they're most in place though, I don't mind consolidating them. There's a lot of repeated tests, like the nullable test for every data type, that probably should be one test iterating through all the data types. 

I'll apply other suggestions, then I'll consolidate tests into one file in a separate commit 👍